lib/index.js

```diff
@@ -44,71 +48,106 @@ export default function runContentfulExport (usageParams) {
     opts.errorLogFile = opts.exportDir + '/contentful-export-' + Date.now() + '.log'
   }
 
-  const clients = createClients(opts)
-  return getFullSourceSpace({
-    managementClient: clients.source.management,
-    spaceId: clients.source.spaceId,
-    maxAllowedLimit: opts.maxAllowedLimit,
-    includeDrafts: opts.includeDrafts,
-    skipContentModel: opts.skipContentModel,
-    skipContent: opts.skipContent,
-    skipWebhooks: opts.skipWebhooks,
-    skipRoles: opts.skipRoles
-  })
-    .then((response) => {
-      if (!opts.downloadAssets) {
-        return response
+  const listrOptions = opts.useVerboseRenderer
```

Comment:
I find this a bit confusing, using **ternary** operator on multiple lines, 
maybe this logic should be moved to a separate function like `createListrOptions(opts.useVerboseRenderer)`
and inside the function, you can use normal `if/else`. This approach is better I think so next time when the options object grows  we won't have an issue understanding what option should go where